app/views/articles/index.html.haml

```diff
@@ -0,0 +1 @@
+%h1 Welcome to PAPs (Passive Aggressive Publishers)
```

Comment:
Why is this view here? And if it is needed why aren't you showing the articles?
IMO it feels out of scope